### PR TITLE
:FC-1097: :zap: fix for issues caused by set-interval-async dependency update to v2

### PR DESCRIPTION
**Pull request by agalagade**
with the update to v2 the `await clearIntervalAsync(timer)` requires to be wrapped inside an immediately-invoke function expression or remove the `await` so that it resolves the timeout it causes because of not finishing the process. for more details see here under **A word of caution** https://www.npmjs.com/package/set-interval-async

### CODE REVIEW

File: src/view/undeploy.view.ts [hunk 1]
```diff
@@ -121,8 +121,10 @@ export class UndeployView {
                     deployed = await waitUntilLambdaIsUndeployed();
                     /* istanbul ignore else */
                     if (deployed) {
-                      await clearIntervalAsync(timer);
-                      resolve();
+                      (async () => {
```
Comment (chrbrt) [inline src/view/undeploy.view.ts:124]: can you add a comment a in the code explaining why this was added?

File: src/view/deploy.view.ts [hunk 2]
```diff
@@ -132,8 +132,10 @@ export class DeployView {
                     deployed = await checkIfLambdaIsDeployed();
                     /* istanbul ignore else */
                     if (deployed) {
-                      await clearIntervalAsync(timer);
-                      resolve();
+                      (async () => {
```
Comment (chrbrt) [inline src/view/deploy.view.ts:135]: can you add a comment a in the code explaining why this was added?

**Review by chrbrt — CHANGES_REQUESTED** [review]

File: src/view/undeploy.view.ts (hunk 1 above)
Comment (agalagade) [reply src/view/undeploy.view.ts]: added the comment

**Comment by chrbrt** [comment]
this closes #31

**Review by chrbrt — APPROVED** [review]

**Pull request merged**